src/main/java/org/opensearch/sdk/handlers/ExtensionsInitRequestHandler.java

```diff
@@ -79,6 +85,10 @@ public InitializeExtensionResponse handleExtensionInitRequest(InitializeExtensio
             extensionsRunner.setEnvironmentSettings(settings);
             extensionsRunner.updateNamedXContentRegistry();
             extensionsRunner.updateSdkClusterService();
+            // Use OpenSearch Settings to update client REST Connections
+            String openSearchNodeAddress = extensionInitRequest.getSourceNode().getAddress().getAddress();
```

Comment:
I don't think we can always assume the Transport address will be the HTTP/REST address.  I'm OK with this for now but think that there needs to be some other way of designating the REST address.  See #782 for continued discussion.